roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,3 @@
 dependencies:
   - pip
+  - logrotate
```

Comment:
So i disagree here - but i defer to experience. 

IMO roles should define the bare minimum of things that you use to configure that piece of the stack. In this case the zuul role should do nothing but configure zuul. You can then define on a per host/group level that i want to have zuul here, and also logrotate that looks for output configured by the zuul piece.

I don't know how likely it is someone will reuse our roles but i'd ideally like to be able to publish a standalone zuul role that could be used by infra or someone else and I don't think it should be a requirement of setting up zuul that you have to have logrotate, or a filebeat, or whatever else we log with. That's not part of a part of setting up zuul - IMO. 

The practical side of thi is that i don't think logrotate should be a dependency here, it should be listed in the roles: for hosts: zuul in install-ci.yml